lib/log/sfFileLogger.class.php

```diff
@@ -129,4 +129,60 @@ public function shutdown()
       fclose($this->fp);
     }
   }
+
+  /**
+   * @param $format
+   * @return false|string
+   */
+  public static function strftime($format)
+  {
+    if (version_compare(PHP_VERSION, '8.1.0') < 0) {
+      return strftime($format);
+    }
```

Comment:
Sorry for being a pain!

Reply:
Fixed - thanks for remember me to setup my IDE correctly next time